### PR TITLE
support authorization setting and separation of docsPath and basePath

### PR DESCRIPTION
**Pull request by lpreterite**
The docsPath maybe not in basePath. This can use in sub-apps.

### CODE REVIEW

File: example/app.js [hunk 1]
```diff
@@ -13,50 +15,64 @@ app.use(function(req, res, next) {
...
+        version: pkg.version,
+        basePath: '/api',
+        resourcePath: '/example',
+        info: {
+            'title': pkg.name,
+            'description': pkg.description,
+            //'termsOfServiceUrl': 'http://helloreverb.com/terms/',
+            'contact': 'glavin.wiechert@gmail.com',
+            'license': 'MIT',
+            'licenseUrl': 'https://github.com/Glavin001/feathers-swagger/blob/master/LICENSE'
```
Comment (ekryski) [inline example/app.js]: We should update the licenseUrl to `https://github.com/feathersjs/feathers-swagger/blob/master/LICENSE

File: example/app.js [hunk 2]
```diff
@@ -13,50 +15,64 @@ app.use(function(req, res, next) {
...
+    .configure(feathersSwagger({
+        docsPath:'/docs',
+        version: pkg.version,
+        basePath: '/api',
+        resourcePath: '/example',
+        info: {
+            'title': pkg.name,
+            'description': pkg.description,
+            //'termsOfServiceUrl': 'http://helloreverb.com/terms/',
+            'contact': 'glavin.wiechert@gmail.com',
```
Comment (ekryski) [inline example/app.js]: We shoudl change this to hello@feathersjs.com

**Comment by ekryski** [comment]
This is looking great @lpreterite! Is it in a working state? If so, let's get this merged! If you can resolve conflicts that would be awesome. 😄

**Comment by daffl** [comment]
Pulled in via #17

**Pull request closed without merging**